modules/delegates.js

```diff
@@ -555,11 +565,26 @@ Delegates.prototype.isLoaded = function () {
 Delegates.prototype.internal = {
 	forgingEnable: function (req, cb) {
 		library.schema.validate(req.body, schema.enableForging, function (err) {
+
 			if (err) {
 				return setImmediate(cb, err[0].message);
 			}
 
-			var keypair = library.ed.makeKeypair(crypto.createHash('sha256').update(req.body.secret, 'utf8').digest());
+			var keypair;
+			if (library.config.forging.force) {
+				keypair = library.ed.makeKeypair(crypto.createHash('sha256').update(req.body.secret, 'utf8').digest());
+			} else {
+				var encryptedSecret, decryptedSecret;
+				if (Array.isArray(library.config.forging.secret)) {
+					encryptedSecret = library.config.forging.secret[0];
```

Comment:
Good point. Let's discuss these points in tomorrow's meeting. So, we can finalize our needs and modify the code accordingly.